hasher-matcher-actioner/hmalib/lambdas/api/lcc.py

```diff
@@ -22,13 +25,24 @@ def to_json(self) -> t.Dict:
         }
 
 
-def get_lcc_api() -> bottle.Bottle:
+@functools.lru_cache(maxsize=None)
+def get_index(storage_path, signal_type):
+    index = LCCIndexer.get_recent_index(storage_path, signal_type)
+    return index
+
+
+def get_lcc_api(storage_path) -> bottle.Bottle:
 
     lcc_api = SubApp()
 
     @lcc_api.get("/", apply=[jsoninator])
     def lcc_hasher() -> LCCResponse:
```

Comment:
I had not noticed this in @Samyakk123's PR. My bad. Can you add a path fragment?

I don't want requests to `/api/lcc/` to have any meaning. 

Use `/api/lcc/recently-seen-preview/`. You'll need to change the first argument to `@lcc_api.get`

Reply:
Also, add a docstring explaining what this method does.

Reply:
Any thoughts on this?